archlinuxcn/nordic-theme/lilac.yaml

```diff
@@ -0,0 +1,13 @@
+maintainers:
+  - github: hamkido
```

Comment:
The user has no public email.

Reply:
I am sorry for my mistake.

> The user has no public email.

I have changed the mailbox to be publicly visible on github.

![2021-03-17-112612_241x29_scrot](https://user-images.githubusercontent.com/43724352/111410425-b40fe100-8713-11eb-9425-7eb7da6c6baa.png)